src/App.tsx

```diff
@@ -15,47 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY_ALPHABET = 'aplhabet';
+const SORT_BY_LENGTH = 'length';
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
```

Comment:
and then your sortField will be restricted by values from the SortField:
```ts
sortField: SortField,

Reply:
agree with [comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1107/files#r1258496393)